contrib/rootfs-builder/Makefile

```diff
@@ -20,7 +20,7 @@ rootfs/%/bin/busybox: downloads/stage3-%-systemd-current.tar.bz2 rootfs-files
 
 .PRECIOUS: rootfs/%/bin/echo
 rootfs/%/bin/echo: rootfs/%/bin/busybox
-	sudo sh -c 'COMMANDS=$$($< --list) || exit 1; for COMMAND in $${COMMANDS}; do \
+	sudo sh -c 'COMMANDS=$$($< --list | grep -v "^busybox$$") || exit 1; for COMMAND in $${COMMANDS}; do \
```

Comment:
Why wxclude it?  It's a valid command in its own right.

Reply:
Because the Makefile will call `ln -rs busybox busybox` and it will fail:

```
sudo sh -c 'COMMANDS=$(rootfs/amd64/bin/busybox --list) || exit 1; for COMMAND in ${COMMANDS}; do \
	test -L "rootfs/amd64/bin/${COMMAND}" || ln -rs rootfs/amd64/bin/busybox "rootfs/amd64/bin/${COMMAND}" || exit; \
done'
ln: failed to create symbolic link 'rootfs/amd64/bin/busybox': File exists
```

The busybox command doesn't need a symlink.